src/test/java/pkgGame/SudokuTest.java

```diff
@@ -18,9 +19,53 @@ public void isValidValue_Test1() {
 		// TODO: Yisi will do this...
 		// test with a valid value
 		
-		Sudoku s = new Sudoku(9);
+		/*
+		int[][] puzzle = { { 5, 3, 0, 0, 7, 0, 0, 0, 0 }, { 6, 0, 0, 1, 9, 5, 0, 0, 0 }, { 0, 9, 8, 0, 0, 0, 0, 6, 0 },
+				{ 8, 0, 0, 0, 6, 0, 0, 0, 3 }, { 4, 0, 0, 8, 0, 3, 0, 0, 1 }, { 7, 0, 0, 0, 2, 0, 0, 0, 6 },
+				{ 0, 6, 0, 0, 0, 0, 2, 8, 0 }, { 0, 0, 0, 4, 1, 9, 0, 0, 5 }, { 0, 0, 0, 0, 8, 0, 0, 7, 9 } };
 		
-		//assertTrue(...);
+		Sudoku s = new Sudoku(puzzle);
+		
+		s.setCells();
+		
+		Sudoku.Cell c = new Sudoku.Cell(0, 8);
+
+		assertTrue(s.isValidValue​(c, 2)); 
+		*/
+		int[][] puzzle = { { 5, 3, 0, 0, 7, 0, 0, 0, 0 }, { 6, 0, 0, 1, 9, 5, 0, 0, 0 }, { 0, 9, 8, 0, 0, 0, 0, 6, 0 },
+				{ 8, 0, 0, 0, 6, 0, 0, 0, 3 }, { 4, 0, 0, 8, 0, 3, 0, 0, 1 }, { 7, 0, 0, 0, 2, 0, 0, 0, 6 },
+				{ 0, 6, 0, 0, 0, 0, 2, 8, 0 }, { 0, 0, 0, 4, 1, 9, 0, 0, 5 }, { 0, 0, 0, 0, 8, 0, 0, 7, 9 } };
+		Sudoku s;
+		
+		try {
+			Class<?> c = Class.forName("pkgGame.Sudoku");
+			Constructor constructor = c.getConstructor(new Class[] { int[][].class });
+			constructor.setAccessible(true);
+			s = (Sudoku) constructor.newInstance(puzzle);
+			
+			Method mSetCells = c.getDeclaredMethod("setCells", new Class[] { int.class });
+			Method mIsValidValue = c.getDeclaredMethod("isValidValue", new Class[] { int.class });
+			
+			mSetCells.invoke(s, null);
+			
+			assertTrue((Boolean)mIsValidValue.invoke(s, 2) == true);

```

Comment:
`isValidValue` is expecting it's first parameter to be a `Sudoku.Cell`, not a `Sudoku` object.